src_py/__init__.py

```diff
@@ -252,6 +258,7 @@ def Overlay(format, size):  # pylint: disable=unused-argument
 
 try:
     import pygame.mixer
+    from pygame.mixer import Channel
```

Comment:
`Sound` is missing from here. It needs a test, stubs, and mention in the docs as well (unless it is intentional?).